xml/System/Delegate+InvocationListEnumerator`1.xml

```diff
@@ -85,8 +85,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Implements IEnumerable.GetEnumerator() to return  'this' as the IEnumerator.</summary>
-        <returns>To be added.</returns>
+        <summary>Implements IEnumerable.GetEnumerator() to return 'this' as the IEnumerator.</summary>
+        <returns>An IEnumerator instance that can be used to iterate through the invocation targets of the delegate.</returns>
```

Comment:
Could you also get up the corresponding PR against dotnet/runtime, to ensure that those XML doc comments are up to date?

It should make it simpler to get those to be the eventual source of truth.